examples/segnet/evaluate.py

```diff
@@ -0,0 +1,79 @@
+from __future__ import division
+
+import argparse
+
+import chainer
+from chainer import cuda
+from chainer.dataset import concat_examples
+import chainer.functions as F
+from chainer import serializers
+
+from chainercv.datasets import camvid_label_names
+from chainercv.datasets import CamVidDataset
+from chainercv.links import SegNetBasic
+
+import numpy as np
+
+
+def main():
+    # This follows evaluation code used in SegNet.
+    # https://github.com/alexgkendall/SegNet-Tutorial/blob/master/
+    # # Scripts/compute_test_results.m
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--gpu', type=int, default=-1)
+    parser.add_argument('--snapshot', type=str)
+    parser.add_argument('--batchsize', type=int, default=24)
+    args = parser.parse_args()
+
+    n_class = 11
```

Comment:
In other file, `n_class = 12` is used. If those have different definitions, it is better to used different names.